libs/ui/controls/TagSelectPicker.js

```diff
@@ -6,12 +6,13 @@ const Tag = ({
   label,
   hasChildren,
   isChecked,
+  isExpanded,
   onCheck,
   onExpand,
 }) => {
   return html`
     <div
-      class="tagselect-item"
+      class="tagselect-item${isExpanded ? ' expanded' : ''}"
```

Comment:
super nit:  put the space after `tagselect-item` instead of before `expanded`?

Reply:
Not that it matters but then that always leaves a space